### PR TITLE
Use KMS for vault token if .app.json is present

### PR DESCRIPTION
**Pull request by plukevdh**
Should leave EB flows untouched in the case that an .app.json file does not exist.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+if [ -f ".app.json" ]
+then
+  VAULT_TOKEN=$(jq -r '.vault_token' .app.json | base64 --decode | aws kms decrypt --ciphertext-blob fileb:///dev/stdin --output text --query Plaintext | base64 --decode)
```
Comment (tecnobrat) [inline entrypoint.sh]: I think you want `export` infront of this.

Comment (plukevdh) [inline entrypoint.sh]: ah, fair

**Comment by plukevdh** [comment]
@articulate/ops what do I need to roll this out, a rebuild of the base images in docker hub?

**Comment by plukevdh** [comment]
oh i see thats jenkinsified

**Pull request merged**